tsl/src/remote/dist_ddl.c

```diff
@@ -368,19 +368,24 @@ dist_ddl_process_drop(const ProcessUtilityArgs *args)
 {
 	DropStmt *stmt = castNode(DropStmt, args->parsetree);
 
+	/* For DROP TABLE and DROP SCHEMA operations hypertable_list will be empty */
 	if (list_length(args->hypertable_list) == 0)
 	{
```

Comment:
If the `hypertable_list` is not empty, and `removeType == OBJECT_TABLE`, what would that mean?

Reply:
Or what does the default branch in the following switch/case means... looks like this needs more assertions/comments.

Reply:
I believe hypertable can't be added into the hypertable list, because drop operation is already happened by PostgreSQL standard hook (src/process_utility.c) at this point and it can't be resolved

Reply:
Should it be an `Assert` then, with this explanation as a comment?